pallets/gear/src/lib.rs

```diff
@@ -809,21 +814,24 @@ pub mod pallet {
                             context,
                             pages_with_data,
                         } => {
-                            let memory_pages = if lazy_pages_enabled {
+                            #[cfg(feature = "lazy-pages")]
```

Comment:
also it would be great if we had chance to avoid matching here features (it makes code harder to understand)

Reply:
Dont actually understand what do you want. It's just two small peaces of code, very simple to understand what do they mean.